apps/web/src/components/Profile/NftGallery/Gallery.tsx

```diff
@@ -0,0 +1,36 @@
+import SingleNFT from '@components/NFT/SingleNFT';
+import type { Nft } from 'lens';
+import type { FC } from 'react';
+import React, { useState } from 'react';
+import { GridContextProvider, GridDropZone, GridItem, swap } from 'react-grid-dnd';
+type Props = {
+  nfts: Nft[];
+};
+
+const Gallery: FC<Props> = ({ nfts }) => {
+  const [items, setItems] = useState(nfts.filter((nft) => !nft?.originalContent?.animatedUrl));
+
+  function onChange(sourceId: any, sourceIndex: any, targetIndex: any) {
+    const result = swap(items, sourceIndex, targetIndex);
+    console.log('🚀 ~ file: Gallery.tsx:20 ~ onChange ~ result', result);
+    return setItems([...result]);
```

Comment:
```suggestion
    return setItems(result);
```
aren't result array already?